fiftyone/utils/eval/coco.py

```diff
@@ -253,6 +255,7 @@ class COCODetectionResults(DetectionResults):
         recall: an array of recall values
         iou_threshs: an array of IoU thresholds
         classes: the list of possible classes
+        recall_sweep (None): an array of recall values of shape ``num_iou x num_classes``
```

Comment:
nit: our [style guide](https://github.com/voxel51/fiftyone/blob/develop/STYLE_GUIDE.md) says line length should be <80.

Unfortunately the code linter in pre-commit hooks doesn't automatically format comments, so we have to manually apply this convention for docstrings/comments.

Adding a vertical ruler to your IDE at 80 characters helps with this.